frontend/src/locale/messages.fr.xlf

```diff
@@ -1570,6 +1572,7 @@
       </trans-unit>
       <trans-unit id="49cef95661d86f4341788ce40068d58801adc6e6" datatype="html">
         <source><x id="START_ITALIC_TEXT" ctype="x-i" equiv-text="&lt;i&gt;"/>There many transactions on this address, more than your backend can handle. See more on <x id="START_LINK" ctype="x-a" equiv-text="&lt;a href=&quot;/docs/faq#address-lookup-issues&quot;&gt;"/>setting up a stronger backend<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a&gt;"/>.<x id="CLOSE_ITALIC_TEXT" ctype="x-i" equiv-text="&lt;/i&gt;"/><x id="LINE_BREAK" ctype="lb" equiv-text="&lt;br&gt;"/><x id="LINE_BREAK" ctype="lb" equiv-text="&lt;br&gt;"/> Consider viewing this address on the official Mempool website instead: </source>
```

Comment:
"There many transactions on this address" shouldn't it be "There **are** many transactions on this address" ?

Reply:
I spotted this typo as well when it came up for something I was searching my Mempool app for.

I signed up an account at transifex.com.  Located the mempool organization there.  Tried to join the organization, but was unable to because it wanted me to specify which language(s) I spoke.  I only speak English (United States).  This typo is in English.  The source language of the Mempool organization is English (United States).  It seems to me that therefore the typo would exist in https://github.com/mempool somewhere, but I was unable to locate it.
![Screenshot from 2022-12-07 21-40-48](https://user-images.githubusercontent.com/967793/206303844-652bb9d2-2190-4b0b-a1e5-7b351cf2036b.png)